imod_coupler/drivers/ribametamod/ribametamod.py

```diff
@@ -469,8 +471,11 @@ def exchange_sprinkling_demand_msw2rib(self, delt: float) -> None:
         self.ribasim_user_demand[:] = (
             masked[:, np.newaxis] * self.ribasim_user_demand[:]
         )
-        self.ribasim_user_demand[:] += mapped[:, np.newaxis] * self.ribasim_userprio
```

Comment:
mapped values are not set into the ribasim user demand var. The line above is serves to keep the values in that var that should not be overwritten, but the remove line is supposed to set the mapped values